lib/ansible/modules/net_tools/nios/nios_network.py

```diff
@@ -87,9 +87,9 @@
 '''
 
 EXAMPLES = '''
-- name: configure a network
+- name: configure a networkv4/v6
   nios_network:
-    network: 192.168.10.0/24
+    network: 192.168.10.0/24 or fe80::/64
```

Comment:
Does this actually work this way with `or`?

Reply:
yes, it works this way also if user gives complete ipv6 address that also works. Coz the feature request was raised with respective example so thought of adding it same way.

Reply:
Yeah, the EXAMPLES field should be a valid, runnable list of plays. If you want to show off different valid inputs, the better way is to have multiple examples

Reply:
I have modified the playbook examples to valid ones, also have added one new networkv6 example